### PR TITLE
Reformat Error Page for Budgets page

### PR DESCRIPTION
**Pull request by lekhacminhphuong**
1/ When there is an error for user's input, the save button navigates user to the error page which do not have the same layout format with other pages of the AQ -> Fixed
2/ Added edit icons for edit actions in the budgets table -> be consistent with other pages.
3/ Updated the input form for create budget (added place holder, required validation,...)

 (still in progress)

### CODE REVIEW

File: app/views/budgets/index.html.erb [hunk 1]
```diff
@@ -6,22 +6,53 @@
 
 <% content_for :sidebar do %>
   <div flex layout-padding>
-    <br><br>
-    <%= render 'form' %>
+    <h3>Create Bugget</h3>
+
+<%= form_for(@budget) do |f| %>
```
Comment (bjkeller) [inline app/views/budgets/index.html.erb]: Is there a reason to move this from the form partial to the index file?

I think we should keep the same organization across the different views, so would be better to keep the forms partial file.

File: app/views/budgets/index.html.erb [hunk 2]
```diff
@@ -6,22 +6,53 @@
...
-        <th>Users</th>      
-        <th></th>
+        <th style="width: 10%">Name</th>
+        <th style="width: 30%">Description</th>
+        <th style="width: 10%">Contact</th>  
+        <th style="width: 40%">Users</th>      
+        <th style="width: 5%"></th>
       </tr>
 
       <% @budgets.each do |budget| %>
```
Comment (bjkeller) [inline app/views/budgets/index.html.erb]: let's use the alphabetical pagination here - sort on budget name

File: app/views/budgets/index.html.erb (hunk 1 above)
Comment (lekhacminhphuong) [reply app/views/budgets/index.html.erb]: That's a good point for consistency.
I think let's put this feedback as resolved and I will create new branch for formatting all the form files.

File: app/views/budgets/index.html.erb (hunk 2 above)
Comment (lekhacminhphuong) [reply app/views/budgets/index.html.erb]: I added the alphabetical pagination

**Pull request merged**